stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/state/ChannelState.kt

```diff
@@ -0,0 +1,83 @@
+package io.getstream.chat.android.offline.experimental.channel.state
+
+import io.getstream.chat.android.client.models.ChannelUserRead
+import io.getstream.chat.android.client.models.Member
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.TypingEvent
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.core.ExperimentalStreamChatApi
+import io.getstream.chat.android.offline.channel.ChannelData
+import kotlinx.coroutines.flow.StateFlow
+
+@ExperimentalStreamChatApi
+/** State container with reactive data of a channel.*/
+public interface ChannelState {
```

Comment:
Does it make sense to mark it with InternalApi for now?

Reply:
Done